_html/scss/modules/_dimonds-sponsors-content.scss

```diff
@@ -0,0 +1,39 @@
+.dimonds-background {
+    background: url('../images/backgrounds/dimond-sponsors-bg.png') no-repeat;
```

Comment:
You already have this image in background as footer-background.png. Please use that so we don't have duplicates.